modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SelectWithOption.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClaySelect, ClaySelectWithOption} from '@clayui/form';
+import React from 'react';
+
+import {FieldBase} from './FieldBase';
+
+interface ISelectWithOptionProps
+	extends React.SelectHTMLAttributes<HTMLSelectElement> {
+	ariaLabel?: string;
+	disabled?: boolean;
+	error?: string;
+	feedbackMessage?: string;
+	label?: string;
+	options: Array<
+		(
+			| React.ComponentProps<typeof ClaySelect.Option>
+			| React.ComponentProps<typeof ClaySelect.OptGroup>
+		) & {
+			options?: Array<React.ComponentProps<typeof ClaySelect.Option>>;
+			type?: 'group';
+		}
+	>;
+	required?: boolean;
+	tooltip?: string;
+}
+
+export function SelectWithOption({
+	ariaLabel,
+	className,
+	disabled,
+	error,
+	feedbackMessage,
+	id,
+	label,
+	onChange,
+	options,
+	required,
+	tooltip,
+	value,
+	...otherProps
+}: ISelectWithOptionProps) {
+	return (
+		<FieldBase
+			className={className}
+			disabled={disabled}
+			errorMessage={error}
+			helpMessage={feedbackMessage}
+			id={id}
+			label={label}
+			required={required}
+			tooltip={tooltip}
+		>
+			<ClaySelectWithOption
+				{...otherProps}
+				aria-label={ariaLabel}
+				onChange={onChange}
+				options={options}
+				value={value}
```

Comment:
The 	`aria-label` one I need to pass like this because of formatting issues (I can't create a prop with the `-` character in `aria-label` so I had to create a `ariaLabel` and pass to the aria-label one. The others I will change!